CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [2.3.0] - 2023-11-23
 ### :magic_wand: Added
+- Add fast switchover support for Amazon RDS Multi-AZ DB Clusters ([PR #690](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/690))
```

Comment:
nit: Adding missing period to make it consistent across the other changes.

```suggestion
- Add fast switchover support for Amazon RDS Multi-AZ DB Clusters ([PR #690](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/690)).
```

Reply:
`Add` is generally omitted in the `Added` section in this file since it is a bit redundant.
```suggestion
- Fast switchover support for Amazon RDS Multi-AZ DB Clusters ([PR #690](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/690)).
```